app/models/channel_queue_membership.rb

```diff
@@ -5,5 +5,18 @@ class ChannelQueueMembership < ActiveRecord::Base
   validates_presence_of :user_id, :channel_queue_id
   validates :user_id, uniqueness: { scope: :channel_queue_id }
 
-  scope :active, -> { where('channel_queue_memberships.created_at >= ?', Time.now.in_time_zone('Pacific Time (US & Canada)').beginning_of_day + 7.hours)}
+  scope :active, -> { where('channel_queue_memberships.created_at >= ?', ChannelQueueMembership.queue_open_time)}
+  scope :inactive, -> { where('channel_queue_memberships.created_at < ?', ChannelQueueMembership.queue_open_time)}
+
+  def active?
+    created_at >= ChannelQueueMembership.queue_open_time
+  end
+
+  def inactive?
+    !active?
+  end
+
+  def self.queue_open_time
+    Time.now.in_time_zone('Pacific Time (US & Canada)').beginning_of_day + 7.hours
```

Comment:
Do you think we need an upper bound too?

Reply:
I don't think so, since this uses `Time.now.in_time_zone.beginning_of_day` it's gonna effectively "clear" from midnight